### PR TITLE
add rsh launch method

### PR DESCRIPTION
**Pull request by andre-merzky**
This is needed for the Allegro cluster at FU Berlin.

### CODE REVIEW

**Comment by marksantcroos** [comment]
Can you remove _which() then?
And have _find_executable() also use ru.which?

**Comment by andre-merzky** [comment]
That makes sense of course, will do.

**Comment by andre-merzky** [comment]
Mark, if you have the time, please look over this PR again.  Thanks!

File: src/radical/pilot/agent/lm/yarn.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 import os
 import subprocess
 
+import radical.utils as ru
```
Comment (marksantcroos) [inline src/radical/pilot/agent/lm/yarn.py]: There is a duplicate import of this now.

Comment (andre-merzky) [inline src/radical/pilot/agent/lm/yarn.py]: thanks, will fix

**Comment by andre-merzky** [comment]
Thanks Mark!

**Pull request merged**